src/services/TestNumberService.ts

```diff
@@ -34,11 +36,12 @@ export class TestNumberService {
                 const testNumber: TestNumber = this.createNextTestNumberObject(lastTestNumber);
                 return this.dbClient.transactWrite(testNumber, lastTestNumber)
                     .then(() => {
+                        console.log("Test Number Generated successfully");
```

Comment:
Not necessary? Will be assumed to be successful unless otherwise stated.

Reply:
This is to add cloudwatch logs. It would help in identifying success/failure rate.